estate_account/models/inherited_estate_model.py

```diff
@@ -0,0 +1,28 @@
+from odoo import models, Command
+
+class InheritedEstateModel(models.Model):
+    _inherit = 'estate.property'
+
+    def sold(self):
+        selling_price = self.selling_price
+        account_move = {
+            'partner_id': self.buyer_id,
+            'move_type': 'out_invoice',
+            "line_ids": [
+                Command.create({
+                    "name": "Selling price",
+                    "quantity": 1,
+                    "price_unit": selling_price * 0.06
+                }),
+                Command.create({
+                    "name": "administrative fees",
+                    "quantity": 1,
+                    "price_unit": 100
+                }),
+            ],
+        }
+        self._create_invoices(account_move)
+        return super(InheritedEstateModel, self).sold()
+
+    def _create_invoices(self, invoice):
+        self.env['account.move'].create(invoice)
```

Comment:
I fear this might not work if the user is not an admin. This technical `create` might require a `sudo()` to work for other users of the `estate` app.

Missing EOL :wink: